.github/workflows/conformance-e2e-v1.13.yaml

```
@@ -236,14 +236,14 @@ jobs:
             lb-acceleration: 'testing-only'
 
           - name: '8'
-            kernel: 'bpf-next-20230420.212204'
+            kernel: 'bpf-next-main'
```

Briefly discussed with @tklauser

```suggestion
            kernel: 'bpf-next-main' # renovate: datasource=docker depName=quay.io/lvh-images/kind sha=sha256:bd6a4dbf830a73c787cc2e4e98db3f8928f494d7e5f22c82f8bb211b78c907b6
```

and then we need something like https://github.com/cilium/cilium-cli/pull/1667/files#diff-20ab1190d08a53a3012bc191ed56205c8f0ffb8fa1715e9d36ecfd1d2ea8a790R159-R168


let's sync next week.

---

@aanm Bump, any action needed from my side?

---

@brb yes, we need something similar as what I suggested